File: src/engine-core/src/project.rs
// Copyright 2020 The Model Authors. All rights reserved.
// Use of this source code is governed by the Apache License,
// Version 2.0, that can be found in the LICENSE file.

use std::collections::HashMap;
use std::rc::Rc;

use super::model;

#[derive(Clone, PartialEq, Debug)]
pub struct Project {
    pub name: String,
    pub models: HashMap<String, Rc<model::Model>>,
}

File: src/app/Project.ts
// Copyright 2021 The Simlin Authors. All rights reserved.
// Use of this source code is governed by the Apache License,
// Version 2.0, that can be found in the LICENSE file.

export interface Project {
  id: string;
  displayName: string;
  description: string;
  tags: string[];
  file: string;
}
